src/factory/mod.rs

```diff
@@ -762,7 +762,13 @@ impl Factory {
                 "Can't read font file:\nFile: {}",
                 file_path.display()
             ));
-        Font::new(buffer, file_path.to_owned(), self.backend.clone())
+        Font::new(buffer, format!("path: {:?}", file_path), self.backend.clone())
+    }
+
+    /// Load the Karla font
+    pub fn load_font_karla(&mut self) -> Font {
+        let buffer = include_bytes!("../../data/fonts/Karla-Regular.ttf").to_vec();
+        Font::new(buffer, String::from("Embeded Karla-Regular.ttf"), self.backend.clone())
```

Comment:
somewhat minor, but we should totally be able to pass in a slice `&[u8]` in here instead of a vector

Reply:
I tried just now but I run into lifetime errors I cant resolve.

Reply:
Shouldn't be possible due to internal organization of rusttype

Reply:
Please elaborate. What's blocking the references here?

Reply:
Okay, our internal code. Static lifetime [here](https://github.com/three-rs/three/blob/master/src/text.rs#L90)

Reply:
Does `GlyphBrush` inherit the lifetime from the raw data we pass in?

Reply:
yep, it does

Reply:
That's the reason I said it's `rusttype` deal mostly. I was forced to use `'static` and `Vec<u8>` while implementing this